specification/maps/data-plane/Render/preview/2.1/render.json

```diff
@@ -0,0 +1,451 @@
+          },
+          {
+            "value": "microsoft.dem",
+            "description": "Digital Elevation Model tiles. The tiles are in the GeoTIFF format with a single 32-bit floating point band. The tiles cover the whole landmass of Earth. Some small islands (e.g., atolls) might not be represented accurately.<br>\n* The vertical unit for measurement of elevation height is meters. An elevation value of -32767.0 is used for points that have no data value, most often returned where there isn't landmass (i.e. water).<br>\n* The horizontal reference datum is the World Geodetic System 1984 (WGS84-G1150) and the vertical reference datum is the Earth Gravitational Model 2008 (EGM2008).<br>\n* Tiles are 258x258 pixel squares rather than the standard 256 x 256. This is done to allow for accurate interpolation of values at the tile edges. As such adjacent tiles overlap by 1 pixel along all edges.<br>\n* Tile data comes from the [Airbus WorldDEM4Ortho product](https://www.intelligence-airbusds.com/worlddem-streaming/). Urban areas are approximately leveled down to ground level. All other areas are represented by the object surface level (e.g., trees). <br>\n\nSupports zoom level 13 only. Format: raster (tiff)."
+          },
+          {
+            "value": "microsoft.dem.contours",
+            "description": "Digital elevation contour line tiles. Compared to the microsoft.dem option, these tiles are in vector format and intended for visualization purpose. The tiles cover the whole landmass of Earth. Some small islands (e.g., atolls) might not be represented accurately.<br>\n* The vertical unit for measurement of elevation height is meters.<br>\n* The horizontal reference datum is the World Geodetic System 1984 (WGS84-G1150) and the vertical reference datum is the Earth Gravitational Model 2008 (EGM2008).<br>\n* Tile data comes from the [Airbus WorldDEM4Ortho product](https://www.intelligence-airbusds.com/worlddem-streaming/). Urban areas are approximately leveled down to ground level. All other areas are represented by the object surface level (e.g., trees).<br>\n\nSupports zoom levels 9 through 14. Format: vector (pbf)."
+          }
+        ]
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "MapTileV2Zoom": {
+      "name": "zoom",
+      "in": "query",
+      "description": "Zoom level for the desired tile. Please find TilesetID list below for more details on supported zoom level for each tilesetId.<br>\n\nPlease see [Zoom Levels and Tile Grid](https://docs.microsoft.com/en-us/azure/location-based-services/zoom-levels-and-tile-grid) for details.",
+      "required": true,
+      "type": "integer",
+      "format": "int32",
+      "x-ms-parameter-location": "method"
+    },
+    "Text": {
```

Comment:
The parameter "Text" is not referenced and I think we could remove it.

Reply:
Thanks, double-checked https://dev.azure.com/msazure/One/_git/Azure-IoT-LocationBasedServices?path=%2Fsrc%2FALSProxy%2FALS%2FModels%2FMap , indeed Text was only used https://dev.azure.com/msazure/One/_git/Azure-IoT-LocationBasedServices?path=%2Fsrc%2FALSProxy%2FALS%2FModels%2FMap%2FCopyrightWorldRequest.cs which belonged to render 1.0 and is not the parameter in new copyright api